pkg/controller/importstatus/importstatus_controller.go

```diff
@@ -104,12 +106,11 @@ func (r *ReconcileImportStatus) Reconcile(ctx context.Context, request reconcile
 		return reconcile.Result{}, err
 	}
 	if !available {
-
-		reqLogger.V(5).Info("Klusterlet manifestworks are not available")
+		reqLogger.Info("Klusterlet manifestworks are not available")
 		return reconcile.Result{}, nil
 	}
 
-	reqLogger.V(5).Info("Klusterlet manifestworks are available")
+	reqLogger.Info("Klusterlet manifestworks are available")
```

Comment:
this should be with higher level, otherwise it would bring noise in the logs.

Reply:
Done.